releasenotes/notes/istiod-helm-args.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: extensibility
+releaseNotes:
+- |
+  **Added** added extra args to istio controller helm chart.
```

Comment:
You don't need the double added.
Capitalize Istio.
Not sure if this is better under installation or extensibility.